frontend/ios/Flutter/Debug.xcconfig

```diff
@@ -1 +1,2 @@
+#include? "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```

Comment:
I assume there are a lot of files which you have not edited yourself. Do these really have to be merged or is this generated code that should be ignored by git? please investigate and verify that